### PR TITLE
Update en.json

### PR DESCRIPTION
**Pull request by sirdavidoff**

### CODE REVIEW

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 1]
```diff
@@ -534,7 +534,7 @@
 		"itLooksLikeYouHaveBeenEditingSomething": "It looks like you have been editing something. If you leave before saving, your changes will be lost.",
 		"loadingTemplate": "Loading template",
 		"moreInfo": "More info",
-		"noNodesGivenToAdd": "No nodes given to add!",
+		"noNodesGivenToAdd": "No to add specified",
```
Comment (ivov) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:537]: You a word.

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 2]
```diff
@@ -773,8 +773,8 @@
 		"noMatchingTagsExist": "No matching tags exist",
 		"noTagsExist": "No tags exist",
 		"showError": {
-			"message": "A problem occurred when trying to create the {name} tag",
-			"title": "New tag was not created"
+			"message": "A problem occurred when trying to create the '{name}' tag",
+			"title": "Couldn't create tag"
```
Comment (ivov) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:777]: For consistency, others above do not use the contraction: `couldn't` -> `could not`

Same for other instances of `couldn't` below.

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 3]
```diff
@@ -1019,18 +1019,18 @@
 			},
 			"saveSettings2": {
 				"errorMessage": "Maximum Timeout is: {hours} hours, {minutes} minutes, {seconds} seconds",
-				"message": "Set timeout is exceeding the maximum timeout!",
+				"message": "The timeout is larger than allowed.",
```
Comment (ivov) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:1022]: `longer`?

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 4]
```diff
@@ -938,71 +938,71 @@
...
+			"message": "If you do this, your current changes will be lost."
 		},
 		"created": "Created",
 		"name": "@:reusableBaseText.name",
 		"openWorkflow": "Open Workflow",
 		"filterWorkflows": "Filter by tags",
 		"searchWorkflows": "Search workflows...",
 		"showError": {
-			"message": "There was a problem loading the workflows",
+			"message": "",
```
[the remark this answers is not in the document]
Comment (sirdavidoff) [reply packages/editor-ui/src/plugins/i18n/locales/en.json:949]: Can these just be deleted from this file, or do other code changes need to happen too?

[the remark this answers is not in the document]
Comment (mutdmour) [reply packages/editor-ui/src/plugins/i18n/locales/en.json:949]: they should be deleted from code as well. can be handled separately if you want to create a ticket for that so this is not blocked

File: packages/editor-ui/src/components/mixins/showMessage.ts [hunk 5]
```diff
@@ -12,13 +12,16 @@ let stickyNotificationQueue: ElNotificationComponent[] = [];
 
 export const showMessage = mixins(externalHooks).extend({
 	methods: {
-		$showMessage(messageData: ElNotificationOptions, track = true) {
+		$showMessage(
+			messageData: Omit<ElNotificationOptions, 'message'> & { message?: string },
```
Comment (mutdmour) [inline packages/editor-ui/src/components/mixins/showMessage.ts:16]: nice

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.164.0`

**Pull request merged**